src/dataset-builder/TestConstants.ts

```diff
@@ -175,3 +175,128 @@ const dummyConcepts = [
+    categories: [
+      { short: 'Female' },
+      { short: 'Male' },
+      { short: 'Other', long: 'Nonbinary, 2 Spirit, Genderqueer, etc.' },
+    ],
+    series,
+    title: 'Gender identity and current age',
+    yTitle: 'AVERAGE AGE',
+    height: '250rem',
+    legendEnabled: false,
+    showSeriesName: false,
+  };
+}
+
+export function generateRandomCohortAgeSeries() {
+  return [{ data: generateRandomNumbers(3, 90) }];
+}
+export function generateRandomCohortAgeData() {
+  return generateCohortAgeData(generateRandomCohortAgeSeries());
+}
+
+export function generateCohortDemographicData(series) {
+  return {
+    categories: [
+      { short: 'Female' },
+      { short: 'Female 18-44' },
+      { short: 'Female 45-64' },
+      { short: 'Female 65+' },
+      { short: 'Male' },
+      { short: 'Male 18-44' },
+      { short: 'Male 45-64' },
+      { short: 'Male 65+' },
+      { short: 'Other', long: 'Nonbinary, 2 Spirit, Genderqueer, etc.' },
+      { short: 'Other 18-44', long: 'Nonbinary, 2 Spirit, Genderqueer, etc. 18-44' },
+      { short: 'Other 45-64', long: 'Nonbinary, 2 Spirit, Genderqueer, etc. 45-64' },
+      { short: 'Other 65+', long: 'Nonbinary, 2 Spirit, Genderqueer, etc. 65+' },
+    ],
+    series,
+    title: 'Gender identity, current age, and race',
+    yTitle: 'OVERALL PERCENTAGE',
+    height: '500rem',
+    legendEnabled: true,
+    showSeriesName: true,
+  };
+}
+
+export function generateRandomCohortDemographicData() {
+  return generateCohortDemographicData(generateDemographicSeries());
+}
+
+export function generateRandomNumbers(numNumbers: number, max: number) {
+  const randomNumbers: number[] = [];
+  for (let i = 0; i < numNumbers; i++) {
+    const randomNumber = Math.floor(Math.random() * max) + 1;
+    randomNumbers.push(randomNumber);
+  }
+  return randomNumbers;
+}
+
+export function generateRandomNumbersThatAddUpTo(total: number, numNumbers: number): number[] {
+  const randomNumbers: number[] = [];
+  let remaining = total;
+  for (let i = 0; i < numNumbers - 1; i++) {
+    const randomNumber = Math.floor(Math.random() * remaining);
+    remaining -= randomNumber;
+    randomNumbers.push(randomNumber);
+  }
+  randomNumbers.push(remaining);
+  return _.shuffle(randomNumbers);
+}
+
+// series and data will always be the same length
+export function addToSeriesData(series: ChartSeries[], data: number[]) {
+  if (series.length !== data.length) throw new Error('series and data must be the same length');
+  for (let i = 0; i < series.length; i++) {
+    series[i].data.push(data[i]);
+  }
+}
+
+export function generateDemographicSeries(): ChartSeries[] {
+  const series: ChartSeries[] = [
+    { name: 'Asian', data: [] },
+    { name: 'Black', data: [] },
+    { name: 'White', data: [] },
+    { name: 'Native American', data: [] },
+    { name: 'Pacific Islander', data: [] },
+  ];
+  // for each of the three gender identity totals,
+  const genderTotals = generateRandomNumbersThatAddUpTo(100, 3);
```

Comment:
Just as a note - and its dummy data so not a big deal, but I think we would want it to add up to the number of participants, not to 100?

That said, I think trying to get the dummy data perfect is not worth the effort, and just letting it render charts seems fine to me.

Reply:
the graph is based on the overall percentage of participants not the number of participants. that's how it was in the mock